File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/UserStatisticsVO.java
package com.sdkwork.app.model;

public class UserStatisticsVO {
    private String createdAt;
    private String updatedAt;
    private String userId;
    private String username;
    private String avatar;
    private String registeredAt;
    private String lastLoginAt;
    private Integer loginCount;
    private Integer totalUsageTime;
    private Integer totalGenerations;
    private Integer totalProjects;
    private Integer totalFiles;
    private Integer totalStorage;
    private UserLevelVO level;
    private VipStatisticsVO vipInfo;
    private AchievementStatisticsVO achievementStats;
    private UsageStatisticsVO usageStats;
    private GenerationStatisticsVO generationStats;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getUserId() {
        return this.userId;
    }
    
    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getUsername() {
        return this.username;
    }
    
    public void setUsername(String username) {
        this.username = username;
    }

    public String getAvatar() {
        return this.avatar;
    }
    
    public void setAvatar(String avatar) {
        this.avatar = avatar;
    }

    public String getRegisteredAt() {
        return this.registeredAt;
    }
    
    public void setRegisteredAt(String registeredAt) {
        this.registeredAt = registeredAt;
    }

    public String getLastLoginAt() {
        return this.lastLoginAt;
    }
    
    public void setLastLoginAt(String lastLoginAt) {
        this.lastLoginAt = lastLoginAt;
    }

    public Integer getLoginCount() {
        return this.loginCount;
    }
    
    public void setLoginCount(Integer loginCount) {
        this.loginCount = loginCount;
    }

    public Integer getTotalUsageTime() {
        return this.totalUsageTime;
    }
    
    public void setTotalUsageTime(Integer totalUsageTime) {
        this.totalUsageTime = totalUsageTime;
    }

    public Integer getTotalGenerations() {
        return this.totalGenerations;
    }
    
    public void setTotalGenerations(Integer totalGenerations) {
        this.totalGenerations = totalGenerations;
    }

    public Integer getTotalProjects() {
        return this.totalProjects;
    }
    
    public void setTotalProjects(Integer totalProjects) {
        this.totalProjects = totalProjects;
    }

    public Integer getTotalFiles() {
        return this.totalFiles;
    }
    
    public void setTotalFiles(Integer totalFiles) {
        this.totalFiles = totalFiles;
    }

    public Integer getTotalStorage() {
        return this.totalStorage;
    }
    
    public void setTotalStorage(Integer totalStorage) {
        this.totalStorage = totalStorage;
    }

    public UserLevelVO getLevel() {
        return this.level;
    }
    
    public void setLevel(UserLevelVO level) {
        this.level = level;
    }

    public VipStatisticsVO getVipInfo() {
        return this.vipInfo;
    }
    
    public void setVipInfo(VipStatisticsVO vipInfo) {
        this.vipInfo = vipInfo;
    }

    public AchievementStatisticsVO getAchievementStats() {
        return this.achievementStats;
    }
    
    public void setAchievementStats(AchievementStatisticsVO achievementStats) {
        this.achievementStats = achievementStats;
    }

    public UsageStatisticsVO getUsageStats() {
        return this.usageStats;
    }
    
    public void setUsageStats(UsageStatisticsVO usageStats) {
        this.usageStats = usageStats;
    }

    public GenerationStatisticsVO getGenerationStats() {
        return this.generationStats;
    }
    
    public void setGenerationStats(GenerationStatisticsVO generationStats) {
        this.generationStats = generationStats;
    }
}

File: sdkwork-app-sdk-typescript/src/types/user-statistics-vo.ts
import type { AchievementStatisticsVO } from './achievement-statistics-vo';
import type { GenerationStatisticsVO } from './generation-statistics-vo';
import type { UsageStatisticsVO } from './usage-statistics-vo';
import type { UserLevelVO } from './user-level-vo';
import type { VipStatisticsVO } from './vip-statistics-vo';

/** 用户统计响应 */
export interface UserStatisticsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 用户ID */
  userId?: string;
  /** 用户名称 */
  username?: string;
  /** 用户头像 */
  avatar?: string;
  /** 注册时间 */
  registeredAt?: string;
  /** 最后登录时间 */
  lastLoginAt?: string;
  /** 总登录次数 */
  loginCount?: number;
  /** 总使用时长（分钟） */
  totalUsageTime?: number;
  /** 总生成次数 */
  totalGenerations?: number;
  /** 总项目数 */
  totalProjects?: number;
  /** 总文件数 */
  totalFiles?: number;
  /** 总存储空间（字节） */
  totalStorage?: number;
  /** 用户等级 */
  level?: UserLevelVO;
  /** VIP信息 */
  vipInfo?: VipStatisticsVO;
  /** 成就统计 */
  achievementStats?: AchievementStatisticsVO;
  /** 使用统计 */
  usageStats?: UsageStatisticsVO;
  /** 生成统计 */
  generationStats?: GenerationStatisticsVO;
}
